cpp/oneapi/dal/backend/primitives/selection/select_by_rows.hpp

```diff
@@ -0,0 +1,81 @@
+/*******************************************************************************
+* Copyright 2021 Intel Corporation
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*******************************************************************************/
+
+#pragma once
+
+#include <type_traits>
+
+#include "oneapi/dal/backend/primitives/selection/select_by_rows_base.hpp"
+
+namespace oneapi::dal::backend::primitives {
+
+#ifdef ONEDAL_DATA_PARALLEL
+
+template <typename Float>
+class select_by_rows : public select_by_rows_base<Float> {
```

Comment:
Maybe rename it to `k_select_by_rows`, because @makart19 introduced `selector` that does different things